client/src/Videocard.js

```diff
@@ -0,0 +1,55 @@
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { faThumbsUp, faThumbsDown } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+library.add(faThumbsUp, faThumbsDown);
+
+function Videocard({ video, onUpVote, onDownVote, onRemove }) {
+  let linkFixed = video.url.replace("watch?v=", "embed/");
```

Comment:
if this line can be done after the fetch, this Videocard component can purely focus on the presentation only.

Image one day you want to replace the Videocard component but you forget to add this line, it would be nice to keep the data logic with set useState, have cleaner separation of logic